src/sort_merge.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""Insertion sort module."""
+import time
+from random import randint
+from collections import deque
+
+
+def sort(lst):
```

Comment:
Considering all of your other sorting algorithms, this should probably have a more unique name like "merge_sort"